dsl_parser/utils.py

```diff
@@ -436,3 +438,24 @@ def add_values_node_description(data):
     for k, v in data.items():
         result[k] = {'values': v}
     return result
+
+
+def get_function(value):
+    """Get a template function if the value represents it"""
+    # functions use the syntax {function_name: args}, or
+    # {function_name: args, 'type': type_name} so let's look for
+    # dicts of length 1 where the only key was registered as a function
+    # or length of 2 where the other key is 'type'
+    if not isinstance(value, dict):
+        return None
+    if 0 < len(value) < 3:
```

Comment:
I think this logic is inverted? This means we'll return no function if we have a length of 1 or 2 where we in fact want the opposite?